packages/xod-client/src/processes/reducer.js

```diff
@@ -13,9 +14,14 @@ const makeProcess = (id, state, action) =>
     state
   );
 
+const isProcess = action =>
+  action.meta &&
+  action.meta.status &&
+  action.type !== MESSAGE_ADD &&
+  action.type !== MESSAGE_DELETE;
```

Comment:
Looks fragile. What’s so special in `MESSAGE_ADD` and `MESSAGE_DELETE`? I wonder are there more bullet-proof methods of detection? Why can’t we rely on the `status` field presence alone?